src/test/java/org/takes/servlet/HttpServletResponseFakeTest.java

```diff
@@ -41,6 +41,27 @@
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public final class HttpServletResponseFakeTest {
+    /**
+     * Set-Cookie header name.
+     */
+    private static final String SET_COOKIE = "Set-Cookie:";
+
+    /**
+     * HTTP/1.1 header name.
+     */
+    private static final String HTTP_1_1 = "HTTP/1.1";
+
+    /**
+     * HTTP/1.1 502 bad gateway.
+     */
+    private static final String HTTP_1_1_502 = "HTTP/1.1 502 Bad Gateway";
+
+    /**
+     * HTTP/1.1 101 custom error message.
+     */
+    private static final String HTTP_1_1_101 =
```

Comment:
@fanifieiev And here you could call thsi constant `INFO`. Also, the code and message does not relate each other: HTTP error code 101 reason phrase is `Switching Protocols`.

Reply:
@paulodamaso OK